i18n/id/white-paper.md

```diff
@@ -0,0 +1,361 @@
+
+## Pengelola NFT
+
+Setelah berhasil mengirimkan paket, manajer/pengelola paket akan menerima NFT untuk membuktikan pekerjaan dan kontribusi mereka. Pemegang NFT ini akan secara otomatis menerima semua hadiah yang terkait dengan paket. Manajer/pengelola paket dapat mentransfer kepemilikan pemeliharaan atas suatu paket ke manajer/pengelola paket lain hanya dengan mentransfer NFT paket.
+Keberhasilan transfer NFT akan menyebabkan pemilik baru secara otomatis menerima hadiah paket di masa mendatang.
+
+Bagian penting dari pembangunan reputasi bergantung pada frekuensi dan kuantitas pengiriman paket berkualitas. NFT yang dikirimkan ke manajer/pengelola paket sebagai bukti pekerjaan mereka dapat digunakan oleh sistem reputasi untuk memperbarui reputasi manajer/pengelola paket dan memberi mereka akses ke bagian protokol yang ditinggikan, seperti yang diputuskan oleh tata kelola tea. Namun, untuk mencegah vektor serangan, seperti anggota komunitas membeli reputasi mereka, transfer NFT pengelola tidak akan menghasilkan transfer reputasi. Reputasi harus tetap terkait langsung dengan karya pengembang tertentu dan tidak boleh dipindahtangankan.
+
+# Token Tea
+
+## Mengamankan Jaringan
+
+Sementara banyak blockchain mungkin muncul sebagai solusi infrastruktur yang efektif dan aman untuk mendukung tujuan tea, kami percaya bahwa pertimbangan yang cermat harus diberikan pada tumpukan teknologi di mana sistem tea dibangun.
+
+Skalabilitas, efektivitas biaya, ESG, dan ekstensibilitas pihak ketiga adalah pertimbangan desain penting yang dapat dilayani dengan lebih baik oleh sistem bukti kepemilikan tea. Dalam proof-of-stake, operator node dan peserta jaringan mempertaruhkan nilai ekonomi dalam bentuk token asli rantai untuk meningkatkan keamanan sistem. Operator node dan peserta jaringan menerima imbalan atas keberhasilan produksi blok yang mematuhi aturan jaringan dan menyertakan informasi transaksi yang akurat. Ketidakaktifan (alias node down) atau aktivitas jahat / salah dihukum dengan menghancurkan sebagian kecil token yang dipertaruhkan melalui pemotongan.
+
+Sistem bukti kepemilikan yang didukung oleh token tea akan memungkinkan pemegang token tea untuk berkontribusi pada keamanan sistem dengan *mengambil* tea dan mendukung pengembang sumber terbuka dengan *menyeduh* tea. Kami sepenuhnya menyadari bahwa faktor ekonomi dapat mencegah beberapa pengembang untuk mengintai atau menyeduh tea; dengan demikian, mengintai dan menyeduh akan tersedia hanya dengan satu lembar daun, denominasi terkecil tea yang mewakili seperseratus juta ($ 10 ^ {{-8} $) tea.
```

Comment:
> ($ 10 ^ {{-8} $)

Something odd happened here. The formula is not displayed properly. Please double-check in the `white-paper.md` for reference.